src/smart-app-banner/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface SmartAppBannerProps {
+    show: boolean;
+    link: string;
```

Comment:
Maybe `href` is a better name for this since it is just the destination

Reply:
updated